packages/third-parties/package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "@next/third-parties",
+  "version": "13.4.5-canary.8",
```

Comment:
adding an `"private": true` if you don't want to publish it for now?

Reply:
Good point, added that.